src/security/acl.c

```diff
@@ -0,0 +1,177 @@
+/*
+ * (C) Copyright 2023 Intel Corporation.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause-Patent
+ */
+
+#include <daos/debug.h>
+#include <daos_errno.h>
+#include <daos_security.h>
+#include "acl.h"
+
+bool
+is_ownership_valid(struct ownership *ownership)
+{
+	if (daos_acl_principal_is_valid(ownership->user) &&
```

Comment:
nit: you could replace "if" with "return" here

Reply:
Fixed